RHDEVDOCS-6023: Installing Argo Rollouts CLI (#76625)
openshift/openshift-docs: modules/gitops-installing-argo-rollouts-cli-using-mac-os.adoc

// Module included in the following assemblies:
//
// * argo_rollouts/using-argo-rollouts-for-progressive-deployment-delivery.adoc
:_mod-docs-content-type: PROCEDURE
[id="gitops-installing-argo-rollouts-using-mac-os_{context}"]

> = Installing Argo Rollouts CLI using Mac OS

Reviewer: ```suggestion
= Installing Argo Rollouts CLI by using Mac OS
```

Author: Fixed.

> For macOS, you can install the Argo Rollouts CLI by using the link:https://brew.sh[Homebrew] package manager.

Reviewer: ```suggestion
If you are a macOS user, you can install the Argo Rollouts CLI by using the link:https://brew.sh[Homebrew] package manager.
```

Author: Fixed.

.Prerequisites

> * You must have Homebrew (`brew`) installed.

Reviewer: Same comment to prereqs as with Linux module.

Reviewer 2: I can confirm this is the only prerequisite to install the CLI.

.Procedure

> * Run the following command to install the Argo Rollouts CLI.

Reviewer: ```suggestion
* Run the following command to install the Argo Rollouts CLI:
```

Author: Fixed.

+
[source,terminal]
----

> $ brew install argoproj/tap/kubectl-argo-rollouts

Reviewer: Any verification steps here? Or command outputs?

Author: As per the walkthrough demo provided by the SME, we did not view any additional output after this step.

Reviewer 2: Correct, I based the structure of this section on https://docs.openshift.com/container-platform/4.15/cli_reference/openshift_cli/getting-started-cli.html#cli-installing-cli-brew_cli-developer-commands
